cmd/plume/prerelease.go

```diff
@@ -436,7 +436,9 @@ func awsUploadToPartition(spec *channelSpec, part *awsPartitionSpec, imageName,
 		foundBucketRegion := false
 		for _, region := range part.Regions {
 			if region != part.BucketRegion {
-				destRegions = append(destRegions, region)
+				if !pv || aws.RegionSupportsPV(region) {
+					destRegions = append(destRegions, region)
+				}
```

Comment:
Perhaps we should have an "else if pv" that logs to indicate the region pv's being skipped for?